source/module_elecstate/elecstate_getters.h

```diff
@@ -0,0 +1,8 @@
+namespace elecstate
+{
+
```

Comment:
Please add some annotations to explain the function of getters.

Reply:
OK, done.